packages/KCardCatalog/KCatalogItem.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <router-link
+    v-if="locationParam"
+    :to="locationParam">
+    <KCard
+      :data-testid="item ? `${item.title.replace(' ', '-')}-catalog-item` : 'catalog-item'"
+      :test-mode="testMode"
+      class="grid-item d-flex flex-column overflow-hidden k-card-catalog-item"
+      has-hover
+      role="button"
+      tabindex="0"
+      @click.native="handleCardClick($event, item)"
+    >
+      <template v-slot:title>
+        <slot name="cardTitle">
+          {{ item.title }}
+        </slot>
+      </template>
+      <template v-slot:body>
+        <div
+          :class="{ 'multi-line-truncate': truncate }">
+          <slot name="cardBody">
+            {{ item.description }}
+          </slot>
+        </div>
+      </template>
+    </KCard>
+  </router-link>
+
+  <KCard
+    v-else
+    :data-testid="item ? `${item.title.replace(' ', '-')}-catalog-item` : 'catalog-item'"
+    :test-mode="testMode"
+    class="grid-item d-flex flex-column overflow-hidden k-card-catalog-item"
+    has-hover
+    role="button"
+    tabindex="0"
+    @click.native="handleCardClick($event, item)"
+  >
+    <template v-slot:title>
+      <slot name="cardTitle">
+        {{ item ? item.title : '' }}
+      </slot>
+    </template>
+    <template v-slot:body>
+      <div
+        :class="{ 'multi-line-truncate': truncate }">
+        <slot name="cardBody">
+          {{ item ? item.description : '' }}
+        </slot>
+      </div>
+    </template>
+  </KCard>
+</template>
+
+<script>
+import KCard from '@kongponents/kcard/KCard.vue'
+
+export default {
+  name: 'KCatalogItem',
+  components: {
+    KCard
+  },
+  props: {
+    item: {
+      type: Object,
+      default: () => {},
+      // Item must have a title
```

Comment:
If we're requiring the title prop, should we add `required: true` here?

Reply:
Let's just remove the requiredness